.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,91 @@
+name: Smoke Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *" # every day at midnight
```

Comment:
Every day is fine but change the time. GitHub struggles when everyone schedules on the hour.

59 minutes past the hour is a good time 😂

Reply:
Oh, wait. These are our own runners :facepalm: